openzeppelin/token/erc721/ERC721_Mintable_Pausable.cairo

```diff
@@ -21,16 +21,16 @@ from contracts.token.ERC721_base import (
     ERC721_setTokenURI
 )
 
-from contracts.ERC165_base import ERC165_supports_interface
+from openzeppelin.ERC165_base import ERC165_supports_interface
 
-from contracts.Pausable_base import (
+from openzeppelin.Pausable_base import (
     Pausable_paused,
     Pausable_pause,
     Pausable_unpause,
     Pausable_when_not_paused
 )
 
-from contracts.Ownable_base import (
+from openzeppelin.Ownable_base import (
```

Comment:
we could follow [the solidity lib](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/access) and move Ownable under `access/`
```suggestion
from openzeppelin.access.ownable import (
```

and we can move the current `Ownable.cairo` implementation to the testing folder where I think it belongs. I wouldn't consider it a preset.